### ISSUE TITLE
Wiring guide

### ISSUE THREAD

**Issue by BrunoB81HK** [philpownall/ESPHomeRoomba#1]
Hi @philpownall, 

Thanks for your great work! I'm planning on using it for an old roomba and I was wondering if you had any schematics or simple wiring guide to help get started?

I have looked at the other/previous repos and they have some sort of wiring guide, but I was not sure if you had to change anything to make the new components work. 

Looking forward to your reply!

**Comment by philpownall** [comment]
Hi Bruno,

No particular wiring guide - I used the diagrams in:
https://community.home-assistant.io/t/add-wifi-to-an-older-roomba/23282/19

Basically, you take TX from the Roomba to the level shifter to change it to 3V, and from there to whatever pin you use for RX on the ESP32.  Be sure to match the pin number in the ESP32 code.

I used a Level converter (rather than a single transistor), and an ESP32 on an Expansion board, so that I had both 3V and 5V supplies for the level shifter.  I used Standard 2.54mm Pitch jumper wires to connect the DIN socket to the level shifter and from there to the ESP32.

I really enjoy driving the Roomba around using the Home Assistant app on my phone, I hope you do to!
